Bottom: c3a8026848c1cd3e19a01c01e207b2f81cc8f356
Top:    86d6c3416f8ff6d1eb2365876212cb7bddc415d6
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-16 00:57:56 +0000

drc: HACK HACK HACK


---

diff --git a/src/find.c b/src/find.c
index 877deb4..e842fd9 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2623,7 +2623,12 @@ IsPolygonInPolygon (PolygonTypePtr P1, PolygonTypePtr P2)
 
               line.Point1.X = v->point[0];
               line.Point1.Y = v->point[1];
-              line.Thickness = 2 * Bloat;
+              line.Thickness = Bloat;
+              /* Another Bloat is added by IsLineInPolygon, making the correct
+               * 2x Bloat. Ideally we would change it there, but doing so
+               * breaks some other DRC checks which rely on the broken behaviour
+               * in IsLineInPolygon.
+               */
               line.Clearance = 0;
               line.Flags = NoFlags ();
               for (v = v->next; v != &c->head; v = v->next)
